### PR TITLE
docs: add docs team to Angular Team section on collaborators page

### PR DESCRIPTION
**Pull request by brandonroberts**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -685,5 +685,48 @@
...
+    "website": "http://silverpath.org",
+    "bio": "Technical content strategist, architect, project manager, and writer.",
+    "group": "Angular",
+    "lead": "dennispbrown"
+  },
+  "kapunahelewong": {
+    "name": "Kapunahele Wong",
+    "picture": "kapunahele.jpg",
+    "twitter": "kapunahele",
+    "bio": "Developer. Angular. Front end stuff.",
```
Comment (kapunahelewong) [inline aio/content/marketing/contributors.json]: Thanks for this, @brandonroberts. Here's my bio:

> Kapunahele is a developer and Angular fan who works on the Angular docs writing guides and developing example apps. She also enjoys Native Hawaiian practices, textile arts, and marveling at little, inconspicuous plants growing in forgotten places outdoors.

**Review by StephenFluin — APPROVED** [review]

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -685,5 +685,48 @@
...
+    "name": "Judy Bogart",
+    "picture": "judy.png",
+    "group": "Angular",
+    "lead": "dennispbrown"
+  },
+  "jenniferfell": {
+    "name": "Jennifer Fell",
+    "picture": "jennifer.jpg",
+    "website": "http://silverpath.org",
+    "bio": "Technical content strategist, architect, project manager, and writer.",
```
Comment (jenniferfell) [inline aio/content/marketing/contributors.json]: Jennifer is a technical content strategist, architect, designer, and writer. As lead of the Angular docs team, she's always interested in learning more about how developers learn and use Angular. Her offline persona is a horsewoman in Idaho.

File: aio/content/marketing/contributors.json [hunk 3]
```diff
@@ -685,5 +685,48 @@
...
+    "picture": "brandonroberts.jpg",
+    "twitter": "brandontroberts",
+    "website": "https://brandonroberts.dev",
+    "bio": "Brandon is a developer and technical writer working on guides, tutorials, application development, and infrastructure for the Angular docs. He is also a maintainer of the NgRx project, building reactive libraries for Angular.",
+    "group": "Angular",
+    "lead": "dennispbrown"
+  },
+  "chembu": {
+    "name": "Sreevani Sreejith",
+    "picture": "sreevani.jpg",
```
Comment (chembu) [inline aio/content/marketing/contributors.json]: bio: Sreevani is a tech writer with prior programming experience. She writes documentation for the Angular framework team. Outside of work, she likes practicing yoga, honing her skills on classical dance forms, and baking cakes.

**Comment by dennispbrown** [comment]
Here's my bio:
Denny is founder of Expert Support, a professional services firm specializing in technical communication, and leads the Angular technical writing team. His lifelong passion has been to reduce the time and effort required to understand complex technical information. Early on, he was Associate Chairman of the Computer Science Department at Stanford, where he taught introductory courses in programming. He also plays old-timers baseball in local leagues and national tournaments.

**Pull request closed without merging**